frontend/src/components/filtered-list.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export function FilteredList(props) {
```

Comment:
Add a jsdoc explaining this function.

This component is also sufficiently high-level that you should include propTypes. Look at https://blog.logrocket.com/validating-react-component-props-with-prop-types-ef14b29963fc/